packages/fuels/src/cli/commands/deploy/index.ts

```diff
@@ -13,9 +16,12 @@ import { deployContract } from './deployContract';
 import { getDeployConfig } from './getDeployConfig';
 import { saveContractIds } from './saveContractIds';
 
-export async function deploy(config: FuelsConfig) {
+export async function deploy(config: FuelsConfig, _program: Command) {
```

Comment:
How come you've added `_program`, seems unused?

Reply:
The [new] method [signature definition](https://github.com/FuelLabs/fuels-ts/pull/1784/files/c2624e51e44b2aa983877ede2815861312c84c5b#diff-004a628a5877d71458702addd56c030e11787ec9dccea78251d8952d2c39e36bR20) requires the param to be defined (previously, it was optional), but since it's not used in _this_ case, the `_` prefix bypasses the lint error.

https://github.com/FuelLabs/fuels-ts/blob/c2624e51e44b2aa983877ede2815861312c84c5b/.eslintrc.js#L63-L68